### ISSUE TITLE
application menu dismissOnClickMobile false doesn't work on mobile

### ISSUE THREAD

**Issue by ahnpnl** [infor-design/enterprise-ng#356]
**Describe the bug**
Try to set dismissOnClickMobile to false but application menu still closes when clicking outside the menu on small screens/mobile screens

**To Reproduce**
1. Start example app inside `enterprise-ng` repo
2. Go to `app.component.html` and modify `[dismissOnClickMobile]` to be `[dismissOnClickMobile]="false"`
3. Open chrome dev tool and use device simulator and choose a phone, e.g iPhone 8
4. Click on hamburger icon to open menu and then click outside the menu. Observe that menu closes.

**Expected behavior**
`dismissOnClickMobile` false value shouldn't close the menu when clicking outside the menu. Or is it a misleading regarding to this option

**Version**
- ids-enterprise-ng: 5.0.0

**Screenshots**
**N.A.**

**Platform**
 - Device: **N.A.**
 - OS Version: Windows 10
 - Browser Name: Chrome
 - Browser Version Version: 72.0.3626.96 (Official Build) (64-bit)

**Additional context**
There should be at least some documentations about this option.

**Comment by tmcconechy** [comment]
I agree this is maybe a misleading option. What it does is close the menu when you select a menu item by clicking it (or not). The menu is always cancelled and dismissed when you click out. Im not sure why you would want to turn off being able to dismiss the menu by clicking out to cancel/remove it.

Maybe we just need to update the doco on this?

Can you move this to EP @clepore  please?

**Comment by ahnpnl** [comment]
Hi Tim,

For an application in ION, we have a scenario
![image](https://user-images.githubusercontent.com/27772165/52719270-96f10d00-2fa5-11e9-916e-1e3b05f9b7a2.png)
this modal dialog is opened by clicking on Settings menu. The requirement says when the users click on the slider inside this modal dialog, the menu shouldn't close.

I'll forward your answer above to the BA. Just let you know there is such a requirement like that.

**Comment by tmcconechy** [comment]
Ok, It probably shouldnt close if clicking anything on the modal. So thats probably an issue.
I think the setting is right, we may just need to ignore any clicks on a modal if those are closing it.

I think we could make a sample to reproduce this. Can you move this over to EP please @clepore ?
